### PR TITLE
Followed SUSI Webchat design and moved drop down menu to the right

### LINKED ISSUE #164: Follow SUSI Webchat design and move drop down menu to the right
SUSI Webchat keeps the drop down menu on the right. Also, the button for night mode should move to the "Themes" button (also needs to be changed in SUSI Web chat).

### PR DESCRIPTION
**Pull request by 0x48piraj**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [ ] My branch is up-to-date with the Upstream `master` branch.
- [ ] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Changes proposed in this pull request:

- Moved the **drop down** to **right** as in **chat.susi.ai**

### CODE REVIEW

File: src/script.js [hunk 1]
```diff
@@ -165,14 +162,14 @@ function composeReplyTable(response,columns,data) {
             if(item[key]){
                 flag=true;
                 var tdItem = document.createElement("td");
-                if(key==="web_pages") {
```
Comment (zamhaq) [inline src/script.js]: @0x48piraj why are you removing these lines?

**Comment by zamhaq** [comment]
@0x48piraj please update your fork.

File: src/script.js (hunk 1 above)
Comment (0x48piraj) [reply src/script.js]: I haven't  :open_mouth:

Comment (0x48piraj) [reply src/script.js]: fixed

Comment (0x48piraj) [reply src/script.js]: @zamhaq Fixed it!

**Pull request merged**